### PR TITLE
Component updates

### PR DESCRIPTION
**Pull request by sebader**

### CODE REVIEW

File: .ado/pipelines/config/configuration.yaml [hunk 1]
```diff
@@ -5,15 +5,15 @@ variables:
...
 - name: 'kubernetesVersion'   # kubernetes version used for aks clusters
   value: '1.22.6'
 - name: 'helmVersion'         # helm package manager version
-  value: 'v3.8.0'
+  value: 'v3.8.1'
 - name: 'ingressNginxVersion' # nginx ingress controller helm chart version
   value: '4.0.16'
 - name: 'certManagerVersion'  # cert-manager helm chart version
-  value: 'v1.6.2'
+  value: 'v1.7.1'
```
Comment (heoelri) [inline .ado/pipelines/config/configuration.yaml:16]: @sebader have you already tested if 1.7.1 works? Had some trouble when upgrading to 1.6.x haven't played with 1.7.x yet.

Comment (sebader) [reply .ado/pipelines/config/configuration.yaml:16]: Yes, worked without issues. What did you see?

Comment (heoelri) [reply .ado/pipelines/config/configuration.yaml:16]: Manifests requiring updates / new api versions used etc. but might be only the case in the last minor release. Haven't read the release notes for any of the 1.7.x releases, yet.

Comment (sebader) [reply .ado/pipelines/config/configuration.yaml:16]: Interesting. It's running in the private-aks branch (with DNS issuer) in this env

**Comment by heoelri** [comment]
Validating the remaining component-updates here: https://dev.azure.com/mission-critical/Connected/_build/results?buildId=838&view=results

**Comment by heoelri** [comment]
Successfully tested component-updates in e2e. https://dev.azure.com/Mission-Critical/Connected/_build/results?buildId=838&view=results

**Review by heoelri — APPROVED** [review]

**Pull request merged**